### DISCUSSION TITLE
The grid loses its selection when changing pagination

### DISCUSSION THREAD

**Discussion by iuriadmconsorcio** [vikramlearning/blazorbootstrap#420] [Q&A]
How do I keep the selected row after changing pagination?
![gridbootstrap](https://github.com/vikramlearning/blazorbootstrap/assets/99689400/bbf78b1e-51ed-4bd3-ab92-5a208aa78cc8)

**Comment by gvreddy04** [comment]
@iuriadmconsorcio Thank you for using **BlazorBootstrap**. Currently, persisting row selection between pages is not supported. However, we can consider default selection while switching between pages. Please share your requirements for a better understanding. Then, create an issue. We will add an enhancement in a future release.
